src/software/thunderscope/thunderscope_config.py

```diff
@@ -51,7 +50,11 @@ def initialize_application():
     app = pyqtgraph.mkQApp("Thunderscope")
 
     # Setup stylesheet
-    apply_stylesheet(app, theme="dark_blue.xml")
+    with open("software/thunderscope/resources/dark_theme.qss", "r") as file:
+        app.setStyleSheet(file.read())
```

Comment:
Is this change necessary? Seems like a lot of files in the repo that don't need to be tracked.

Reply:
I've always had qt theme issues on my laptop and this fixes it by providing all the qt assets exp

Reply:
ya maybe a separate PR. I think it would also make more sense to not have this in our repo, instead, download the files to somewhere in `tbotspython` directory.